action.yaml

```diff
@@ -65,6 +65,11 @@ runs:
         mv "${{ github.action_path }}/shellcheck-${scversion}/shellcheck" \
           "${{ github.action_path }}/shellcheck"
 
+    - name: Display shellcheck version
+      shell: bash
+      run: |
+        "${{ github.action_path }}/shellcheck" --version
```

Comment:
Since composite actions are not named during the run, I wonder if it makes sense to assign the value to a variable, and then echoing it with a "Shellcheck version: " prefix?

Reply:
On my terminal the output is

```
ShellCheck - shell script analysis tool
version: 0.7.1
license: GNU General Public License, version 3
website: https://www.shellcheck.net
```

Is it okay?

Reply:
Oh yeah, that's perfect :+1:  
I thought it only showed the version number